### PR TITLE
add new map types

### PR DESCRIPTION
**Pull request by valerijSamsonov**
add 2GIS (https://2gis.ru) and MapsMe(https://maps.me)

### CODE REVIEW

**Comment by mattermoran** [comment]
Hi @valerijSamsonov 
I've tested those maps on both android and ios and they seem to open just fine.
There are couple comments I have.
First I see that you used Russian name for 2GIS and you also renamed some other ones in your last commit. This plugin is used by people from different countries so we would like to keep everything in English and if developers need to support other languages then they could use internationalization library or some other methods. 
And second could you squash your commits into one or two so we can keep the history clean but if you want to keep your history as is then I will squash them on my side before merging.
Cheers

File: android/src/main/kotlin/com/alexmiller/map_launcher/MapLauncherPlugin.kt [hunk 1]
```diff
@@ -33,12 +33,15 @@ class MapLauncherPlugin(private val context: Context, private val activity: Acti
   }
 
   private val maps = listOf(
-    MapModel(MapType.google, "Google Maps", "com.google.android.apps.maps"),
+    MapModel(MapType.google, "Карты Google", "com.google.android.apps.maps"),
```
Comment (mattermoran) [inline android/src/main/kotlin/com/alexmiller/map_launcher/MapLauncherPlugin.kt]: Need to use English

File: android/src/main/kotlin/com/alexmiller/map_launcher/MapLauncherPlugin.kt [hunk 2]
```diff
@@ -33,12 +33,15 @@ class MapLauncherPlugin(private val context: Context, private val activity: Acti
...
     MapModel(MapType.amap, "Amap", "com.autonavi.minimap"),
     MapModel(MapType.baidu, "Baidu Maps", "com.baidu.BaiduMap"),
     MapModel(MapType.waze, "Waze", "com.waze"),
-    MapModel(MapType.yandexNavi, "Yandex Navigator", "ru.yandex.yandexnavi"),
-    MapModel(MapType.yandexMaps, "Yandex Maps", "ru.yandex.yandexmaps")
+    MapModel(MapType.yandexNavi, "Яндекс.Навигатор", "ru.yandex.yandexnavi"),
+    MapModel(MapType.yandexMaps, "Яндекс.Карты", "ru.yandex.yandexmaps"),
+    MapModel(MapType.doubleGis, "2ГИС", "ru.dublgis.dgismobile"),
```
Comment (mattermoran) [inline android/src/main/kotlin/com/alexmiller/map_launcher/MapLauncherPlugin.kt]: Same here. Stick to English

File: android/src/main/kotlin/com/alexmiller/map_launcher/MapLauncherPlugin.kt (hunk 1 above)
Comment (mattermoran) [inline android/src/main/kotlin/com/alexmiller/map_launcher/MapLauncherPlugin.kt]: Need to use English

File: ios/Classes/SwiftMapLauncherPlugin.swift [hunk 3]
```diff
@@ -38,15 +40,18 @@ class Map {
 }
 
 let maps: [Map] = [
-    Map(mapName: "Apple Maps", mapType: MapType.apple, urlPrefix: ""),
-    Map(mapName: "Google Maps", mapType: MapType.google, urlPrefix: "comgooglemaps://"),
+    Map(mapName: "Карты Apple", mapType: MapType.apple, urlPrefix: ""),
+    Map(mapName: "Карты Google", mapType: MapType.google, urlPrefix: "comgooglemaps://"),
```
Comment (mattermoran) [inline ios/Classes/SwiftMapLauncherPlugin.swift]: Change to Engish

File: ios/Classes/SwiftMapLauncherPlugin.swift [hunk 4]
```diff
@@ -38,15 +40,18 @@ class Map {
...
     Map(mapName: "Amap", mapType: MapType.amap, urlPrefix: "iosamap://"),
     Map(mapName: "Baidu Maps", mapType: MapType.baidu, urlPrefix: "baidumap://"),
     Map(mapName: "Waze", mapType: MapType.waze, urlPrefix: "waze://"),
-    Map(mapName: "Yandex Navigator", mapType: MapType.yandexNavi, urlPrefix: "yandexnavi://"),
-    Map(mapName: "Yandex Maps", mapType: MapType.yandexMaps, urlPrefix: "yandexmaps://")
+    Map(mapName: "Яндекс.Навигатор", mapType: MapType.yandexNavi, urlPrefix: "yandexnavi://"),
+    Map(mapName: "Яндекс.Карты", mapType: MapType.yandexMaps, urlPrefix: "yandexmaps://"),
+    Map(mapName: "2ГИС", mapType: MapType.doubleGis, urlPrefix: "dgis://"),
```
Comment (mattermoran) [inline ios/Classes/SwiftMapLauncherPlugin.swift]: Change to English

**Comment by Lomski** [comment]
Any updates for this PR? Would be great to have 2GIS in this package.

**Comment by mattermoran** [comment]
@Lomski support for 2gis is added in 0.12.0

**Pull request closed without merging**